scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -82,7 +87,7 @@
             <td>
               {{ variant.position }}
               {% if case.bam_files %}
-                <a class="btn btn-default btn-sm float-right" href="{{ url_for('alignviewers.igv', sample=case.sample_names, build=case.genome_build, bam=case.bam_files, bai=case.bai_files, contig=variant.chromosome, start=(variant.position - 50), stop=(variant.position + 50), center_guide="T") }}" target="_blank">IGV viewer</a>
+                <a class="btn btn-outline-secondary btn-sm float-right" href="{{ url_for('alignviewers.igv', sample=case.sample_names, build=case.genome_build, bam=case.bam_files, bai=case.bai_files, contig=variant.chromosome, start=(variant.position - 50), stop=(variant.position + 50), center_guide="T") }}" target="_blank">IGV viewer</a>
```

Comment:
This presumably needs an update from the other variants views where checking for file existence changed.

Reply:
(And the current master alignviewers view needs a slight change - it was perhaps not tested with trios?)

Reply:
The error was in the alignviewers where I was checking for the existence of of the track files actually